build/config/PublishData.json

```diff
@@ -13,6 +13,13 @@
             "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "publish-test" ]
+        },
+        "features/readonly-ref": {
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.6.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
```

Comment:
Leave the vsix empty for now.

``` json
"vsix": [ ]
```

Reply:
will it publish the vsix without that?

Reply:
I've put the "vsix" back. We would like to see vsix published as well for this branch.